examples/00_getting_started.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# Getting started with `skore`\n",
+    "\n",
+    "This guide showcases the features of `skrub`, an open-source package that aims at enable data scientist to:\n",
+    "1. Store objects of different types from their Python code: from python lists to scikit-learn fitted pipelines and plotly figures.\n",
+    "2. They can track and visualize these stored objects on a dashboard.\n",
+    "3. This dashboard can be exported into a HTML file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1",
+   "metadata": {},
+   "source": [
+    "## Initialize a Project and launch the UI\n",
+    "\n",
+    "From your shell, initialize a `skore` Project called `project.skore` that will be in your current working directory:\n",
+    "```bash\n",
+    "$ python -m skore create 'project.skore'\n",
+    "```\n",
+    "This will create a skore project directory named `project.skore` in the current directory.\n",
+    "\n",
+    "Then, from your Python code (in the same directory), load the project and store an integer for example:\n",
+    "```bash\n",
+    "$ python -m skore launch project.skore\n",
```

Comment:
Call is not consistent with above call (see similar comment in README)

Reply:
I don't understand, is it still the case now?

Reply:
I added double quotes